app/src/main/java/de/westnordost/streetcomplete/quests/QuestModule.java

```diff
@@ -40,6 +41,7 @@
 				new AddRoadSurface(o),
 				new AddBusStopShelter(o),
 				new AddToiletsFee(o),
+				new AddBabyChangingTable(o),
```

Comment:
Can I change babies at this table? Isn't it called **diaper** changing table? :-o